motor/render/vulkan_renderer.cpp

```diff
@@ -68,55 +152,27 @@ vk::Instance CreateInstance()
                          "Couldn't createInstance");
 }
 
-vk::PhysicalDevice SelectPhyiscalDevice(const vk::Instance& vk_inst)
+vk::PhysicalDevice SelectPhyiscalDevice(const vk::Instance& vk_inst,
```

Comment:
i suppose this should return an optional<phydev> instead, as none of the existing devices might be suitable